src/crypto/tls/ech_test.go

```diff
@@ -1016,3 +813,39 @@ func TestECHHandshake(t *testing.T) {
 		})
 	}
 }
+
+func TestUnmarshalConfigs(t *testing.T) {
```

Comment:
Do we have tests for the handle marshal/unmarshal logic? (It's trivial, but, you know.)

Reply:
Yes, see the end of ech_test.go. They could be more coimplete, I think ... @Lekensteyn recently found a bug here, so it's a good question.

Reply:
👍 Cool beans